internal/recert/recert.go

```diff
@@ -96,3 +121,153 @@ func getIngressCNAndFile(certsFolder string) (string, string, error) {
 	}
 	return "", "", fmt.Errorf("failed to find ingress key file")
 }
+
+func ForceExpireSeedCrypto(recertContainerImage, authFile string, ops ops.Ops) error {
```

Comment:
Can we make it interface? 
Maybe it was a mistake to ask to move it here to say the truth (sorry).